numpy/compat/py3k.py

```diff
@@ -53,6 +53,9 @@ def asstr(s):
 def isfileobj(f):
     return isinstance(f, (io.FileIO, io.BufferedReader, io.BufferedWriter))
 
+def isfilelikeobj(f):
```

Comment:
This function is poorly named - the behavior "is like a readable file opened in binary mode".

Since that behavior is harder to describe than `hasattr(f, 'readinto')`, I think it would be better to use `hasattr` directly at the caller.

Reply:
Fixed. Replaced with `hasattr` directly at the caller.